rules/kubecf/image_list.rb

```diff
@@ -0,0 +1,285 @@
+# frozen_string_literal: true
```

Comment:
Our plan to deprecate bazel doesn't guarantee that this file won't diverge from this: https://github.com/cloudfoundry-incubator/kubecf/blob/master/rules/kubecf/image_list.tmpl.rb

I know this PR was created as a replacement to bazel, but until bazel is gone we need to make sure both build targets work the same. Still, I don't have a suggestion on how to ensure this. Maybe we should just keep that in mind when reviewing PRs.

Reply:
> Our plan to deprecate bazel doesn't guarantee that this file won't diverge

No, it doesn't, but it seems rather unlikely that we will make any changes to the script within the next 4 weeks or so. I don't think this is important.

The file will need to move to a new location though, and I'm not sure if it should go directly into `scripts/`, or into a subdirectory, as you are not expected to call it directly.

Reply:
Agreed, let's keep it where it is for now.